wait_for/__init__.py

```diff
@@ -6,6 +6,8 @@
 from datetime import datetime, timedelta
 from functools import partial
 from threading import Timer
+import types
```

Comment:
Can you directly import `LambdaType` instead of the entire module?

Reply:
Done